machine.go

```diff
@@ -663,6 +664,11 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 		return -1, err
 	}
 
+	/* Ensure systemd-resolved binary is available */
+	if _, err := exec.LookPath("systemd-resolved"); err != nil {
```

Comment:
systemd-reslolved isn't expected to be in the path really; we hardcode `/lib/systemd` elsewhere so you can just check if `/lib/systemd/systemd-resolved` exists (and optionally if it's executable) which will work on both merged and non-merged user systems

Reply:
right I checked there first and fallback to checking $PATH